lib/fedex_rest_api/tracking_object.rb

```diff
@@ -0,0 +1,25 @@
+require "fedex_rest_api/base"
+
+class FedexRestApi::TrackingObject
+  attr_reader :access_token, :include_detailed_scans, :trackable_numbers
+
+  def initialize(tracking_params)
+    @access_token = tracking_params[:access_token] || ""
+    @include_detailed_scans = tracking_params[:include_detailed_scans] || false
+    @trackable_numbers = tracking_params[:trackable_numbers] || []
```

Comment:
It's probably ok not to have a fallback for the token. It'll either be valid, invalid, or nil, and all those cases should be managed by our existing logic.